robottelo/api/utils.py

```diff
@@ -41,18 +41,23 @@ def enable_rhrepo_and_fetchid(basearch, org_id, product, repo,
     return result[0].id
 
 
-def promote(content_view_version, environment_id):
+def promote(content_view_version, environment_id, force=False):
```

Comment:
Usually one promotes content views to lifecycle environments that are in sequence. For example, `Library > Lifecycle Environment 1 > Lifecycle Environment 2 > Lifecycle Environment N`. However, one can force the promotion to any given lifecycle environment, even if said lifecycle environment is 'out of sequence'.